src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -1,18 +1,27 @@
-import React, { ChangeEventHandler, FC, FocusEventHandler, KeyboardEventHandler, useEffect, useReducer } from 'react'
+import React, {
```

Comment:
why is formatting messed up? any ideas?

Reply:
I feel there is something going on with the pre-commit hooks, since we introduced lerna...